civis/io/_tables.py

```diff
@@ -422,8 +432,12 @@ def csv_to_civis(filename, database, table, api_key=None,
                   column_delimiter=delimiter, first_row_is_header=headers)
 
     import_job = client.imports.post_files(**kwargs)
-    with open(filename, "rb") as data:
-        put_response = requests.put(import_job.upload_uri, data)
+    if hasattr(filename_or_buf, 'read'):
+        put_response = requests.put(import_job.upload_uri, filename_or_buf)
+    else:
+        with open(filename_or_buf, "rb") as data:
+            put_response = requests.put(import_job.upload_uri, data)
+
```

Comment:
Instead of making a breaking change (i.e. change parameter `filename` to `filename_or_buf`), what do think about creating another function like `_import_bytes` that both `dataframe_to_civis` and `csv_to_civis` call?  We could avoid the somewhat awkward `if` / `else` here.